grails-app/conf/UrlMappings.groovy

```diff
@@ -62,7 +62,7 @@ class UrlMappings {
 		"/api/v1/modules"(resources: "moduleApi")
 		"/api/v1/modules/$id/help"(controller: "moduleApi", action: "help")
 
-		"/api/v1/users/me"(controller: "userApi", action: "getUserInfo")
```

Comment:
Btw have you tested that the `getUserInfo` endpoint works in practice after this URL mapping change? I have a suspicion that you may need to leave in `"/api/v1/users/me"(controller: "userApi", action: "getUserInfo")` because the `(resources: "userApi")` will only map the basic CRUD endpoints (index, create, save, show, edit ... etc. https://grails.github.io/grails2-doc/2.5.6/guide/single.html#restfulMappings) . I may be wrong here, just making sure.